localstack/services/awslambda/lambda_api.py

```diff
@@ -72,6 +71,7 @@
     to_str,
     unzip,
 )
+from localstack.utils.container_networking import get_main_container_name
 from localstack.utils.docker_utils import DOCKER_CLIENT
 from localstack.utils.functions import run_safe
 from localstack.utils.http import canonicalize_headers, parse_chunked_data
```

Comment:
```suggestion
from localstack.utils import container_networking, docker_utils, functions, http
```

Not blocking anything. Maybe we want to go over util names at some point again to make sure they're properly named for using them in such a manner before refactoring all util imports :smile: